.gitignore

```diff
@@ -1,2 +1,3 @@
 /promlens
 /npm_licenses.tar.bz2
+.idea
```

Comment:
Hmm, let's either remove this unrelated change from the PR or end the file with a new line at least.

Reply:
If you don't mind I'd love to keep it. I like the project and would be glad to contribute more, this is just an IDE settings folder.

Reply:
yeah but Prometheus never ignore IDE settings folder like IntelliJ or Eclipse.
Instead you will have to use a global `.gitignore` file, like that: https://stackoverflow.com/questions/7335420/global-git-ignore

For example on my local environment:

```bash
cat .gitconfig
[pull]
	ff = only
[user]
	name = Augustin Husson
	email = husson.augustin@gmail.com
[credential]
	helper = store --file ~/.git/credential
[pager]
	branch = false
[core]
	excludesFile = ~/.git/.gitignore
```

``` bash
cat .git/.gitignore
.idea/
```

Reply:
TIL about global .gitignore file. Thanks! updated.